index.html

```diff
@@ -40,5 +40,209 @@ <h1>I'm Benjamin Glad to see you!</h1>
             </div>
         </section>
     </div>
+    
+    <section class="middle-section-container">
+        <div class="image-area">
+          <img src="photo/logos/Snapshoot-Portfolio.png" alt="">
+        </div>
+
+      <div class="hero">
+          <div class="tonic">
+              <h1>Tonic</h1>
+            <div class="canopy-wrapper">
+              <p>CANOPY</p>
+              <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
```

Comment:
- [  ] As I can see here the font weight and size, color and font family are not the same so kindly Fix them to be the same as the Figma design because it is from the requirements to match the Figma design as much as possible👌🏻👌🏻